### ISSUE TITLE
bring back the old interface

### ISSUE THREAD

**Issue by dfa56** [macchrome/winchrome#147]
Hello,
--disable-features=CustomizeChromeSidePanel does not work in 126.0.6478.103-1_Win64.7z (64-bit).
Please bring back the old interface! If possible.

**Comment by Vladinator** [comment]
The latest version that uses the old interface to my knowledge is [v122](https://github.com/macchrome/winchrome/releases/tag/v122.6261.129-M122.0.6261.129-r1250580-Win64).

Not to steal your issue, but to ask a follow up question. I am not sure when this feature flag got removed, but ever since v123 and later the default is the new look and feel is what I noticed recently.
I don't think I tested to set this particular feature flag, but, if I use v123, v124 or v125, will this flag revert the look back to the same as in v122?

Still, this is a short lived fix since v126 and later will not have this flag unless added as a patch. That can be difficult to maintain if chromium itself changes its code substantially.
